### ISSUE TITLE
Gentoo package

### ISSUE THREAD

**Issue by TheEvilSkeleton** [xanmod/linux#49]
Hello, we have created a Gentoo ebuild for XanMod in our overlay:

https://gitlab.com/src_prepare/src_prepare-overlay

Where the package is located at:

https://gitlab.com/src_prepare/src_prepare-overlay/-/tree/master/sys-kernel/xanmod-sources

the package name is `sys-kernel/xanmod-sources`.

**Comment by hedmo** [comment]
nice to see more gentoo users running xanmod-sources

https://github.com/hedmo/hedmos-overlay

**Comment by TheEvilSkeleton** [comment]
@hedmo That's a really good overlay right here. It seems like your overlay isn't in Layman, nor in eselect-overlay. Would you want to join our group? We have our own Layman and eselect-reporitory repositories so it would easier for your packages to be featured in https://gpo.zugaina.org/.

**Comment by hedmo** [comment]
@TheEvilSkeleton.  of course.i would be happy to join.
@xanmod how about adding gentoo to your homepage?i hanve been using xanmod-sources
sense almost day one from: https://forums.gentoo.org/viewtopic-t-1106938.html and i have to say 
grate job

**Comment by TheEvilSkeleton** [comment]
Here's the Telegram link: https://t.me/joinchat/Mjt_wUusNGlvMJUHRenLBQ

I hope you have a Telegram account.

**Comment by xgqt** [comment]
> Here's the Telegram link: https://t.me/joinchat/Mjt_wUusNGlvMJUHRenLBQ
> 
> I hope you have a Telegram account.

Maybe we will have XMPP in the near future :D

**Comment by xanmod** [comment]
Which xanmod-sources package can I add to the homepage?

**Comment by TheEvilSkeleton** [comment]
Sorry for the late reply, we were still deciding on it. Turns out you have decided to choose, so we will leave at that. Thank you for adding it.
